### PR TITLE
PGE: added upgrade tutorials, supported extensions page, and reorganized left nav bar

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## What Changed?

A request has been made to add documentation to the PGE section since it is rather "lean" compared to the EPAS documentation. 

In this first phase, I have added the following: 
 
- Major upgrade tutorial example with pg_upgrade
- Minor upgrade tutorial example for RPM packages
- Minor upgrade tutorial example for Deb packages
- Supported Extensions page
- Reorganized left-hand nav bar 

Jiras: 
https://enterprisedb.atlassian.net/browse/DOCS-736
https://enterprisedb.atlassian.net/browse/DOCS-735

### CODE REVIEW

File: product_docs/docs/pge/16/upgrading/minor_upgrade/index.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+title: "Minor EDB Postgres Extended Server upgrade"
+navTitle: "Minor upgrades"
+description: "Minor version upgrade of EDB Postgres Extended Server"
+---
+
+Performing a minor upgrade of your EDB Postgres Extended Server is very simple: update your packages and restart the server.
+
+To update your packages, ensure you use the correct package manager for your operating system. 
+
+- If you installed an RPM package of EDB Postgres Extended Server (on RHEL, AlmaLinux, Rocky Linux) with `yum` or `dnf`, see [Minor EDB Postgres Extended Server upgrade of RPM packages](dnf_upgrade).
```
Comment (nidhibhammar) [inline product_docs/docs/pge/16/upgrading/minor_upgrade/index.mdx:11]: We can remove references of yum from the docs as RHEL/CentOS 7 are now EOL

**Review by nidhibhammar — APPROVED** [review]
Rest looks good to me

**Review by ebgitelman — COMMENTED** [review]
Committing this batch but will also edit directly in the branch.

File: product_docs/docs/pge/16/upgrading/major_upgrade.mdx [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    ```
+    mkdir /var/lib/edb-pge/16/upgrade_target 
+    ```
+
+1.  As root, ensure the postgres user owns the directory:
+
+    ```
+    sudo chown enterprisedb /var/lib/edb-pge/16/upgrade_target
+    sudo chgrp /var/lib/edb-pge/16/upgrade_target
+    ```
```
Comment (tusharahuja) [inline product_docs/docs/pge/16/upgrading/major_upgrade.mdx:60]: please replace "enterprisedb" with "postgres" i.e 

sudo chown postgres /var/lib/edb-pge/16/upgrade_target

File: product_docs/docs/pge/16/upgrading/major_upgrade.mdx [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+    This command initializes a CONFIG directory with all configuration files for the encrypted server.
+
+1.  Before you start the cluster, ensure the new database runs on a different port from the source server. To alter the port, edit `postgresql.conf` by uncommenting the line with `#port` and changing the port number, for example, to 5432.
+
+1.  Start the target server: 
+
+    ```
+    /usr/lib/edb-pge/16/bin/pg_ctl -D /var/lib/edb-pge/16/TDE start
+    ```
```
Comment (tusharahuja) [inline product_docs/docs/pge/16/upgrading/major_upgrade.mdx:76]: the data directory name is upgrade_target and not TDE 

 /usr/lib/edb-pge/16/bin/pg_ctl -D /var/lib/edb-pge/16/upgrade_target start

File: product_docs/docs/pge/16/upgrading/major_upgrade.mdx [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+    !!!note 
+       You can also start the server with the `logfile` option enabled to print errors into a logfile: 
+       `/usr/lib/edb-pge/16/bin/pg_ctl -D /var/lib/edb-pge/16/TDE -l logfile start` 
+
```
Comment (tusharahuja) [inline product_docs/docs/pge/16/upgrading/major_upgrade.mdx:81]: same, data directory name is upgrade_target

File: product_docs/docs/pge/16/upgrading/major_upgrade.mdx [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+    ```
+    /usr/lib/edb-pge/16/bin/pg_upgrade -b /usr/lib/edb-pge/15/bin -B /usr/lib/edb-pge/16/bin -d /var/lib/edb-pge/15/upgrade-source -D /var/lib/edb-pge/16/upgrade-target
+    ```
+
+1.  Restart the target server: 
+
+    ```
+    /usr/lib/edb-pge/16/bin/pg_ctl -D /var/lib/edb-pge/16/upgrade-target start
+    ```
```
Comment (tusharahuja) [inline product_docs/docs/pge/16/upgrading/major_upgrade.mdx:125]: not restart but start the server 
i.e  Start the target server

File: product_docs/docs/pge/16/extensions.mdx [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+title: Postgres extensions supported in EDB Postgres Extended Server 
+---
+
+EDB provides support for several Postgres extensions on EDB Postgres Extended Server:
+
+- Open-source extensions
+- EDB supported open-source extensions
+- EDB-developed extensions 
+
+See [Postgres extensions available by deployment](/pg_extensions/) for an overview of all supported extensions and links to their documentation sites.
```
[the remark this answers is not in the document]
Comment (gvasquezvargas) [reply product_docs/docs/pge/16/extensions.mdx:12]: It works in the preview 🥳  

And I think going to the top of the tree is correct here, as pg_extentions has its own section nestled under docs. Is that what you mean?

**Review by theadamwright — APPROVED** [review]

**Pull request merged**